ensembl/htdocs/info/genome/variation/species/species_detailed_counts.html

```diff
@@ -1532,9 +1532,9 @@ <h2 id="prediction" style="margin-top:40px">Prediction data</h2><p style="paddin
          </div>
       </td>
       <td style="text-align:right"><span class="vdoc_var_count vdoc_million_2" title="Over 7.8 million variants with sift data">7.8 M</span></td>
-      <td style="text-align:right"><span style="color:#090" title="Over 2,000 more variants with SIFT predictions"><small>(+2,000)</small></span></td>
-      <td style="text-align:right"><span class="vdoc_var_count vdoc_million_2" title="Over 7.9 million variants with polyphen data">7.9 M</span></td>
-      <td style="text-align:right"><span style="color:#090" title="Over 9,000 more variants with polyphen datas"><small>(+9,000)</small></span></td>
+      <td style="text-align:right"><span style="color:#090" title="Over 26,000 more variants with sift datas"><small>(+26,000)</small></span></td>
```

Comment:
The previous version (`SIFT predictions`) was more appropriate. Is there a reason for this change?

```suggestion
      <td style="text-align:right"><span style="color:#090" title="Over 26,000 more variants with SIFT predictions"><small>(+26,000)</small></span></td>
```